### PR TITLE
fix: stringifying session ID for airship

### PR DESCRIPTION
**Pull request by shrouti1507**
## What are the changes introduced in this PR?

We solved the uuid formatting problem for session ID in another task. But as mobile SDKs send the session ID as a number, `isDefinedAndNotNullAndNotEmpty(1732129099)` was returning false, so the UUID is not being calculated only.

## What is the related Linear task?

Resolves INT-2948

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [x] My code follows the style guidelines of this project

- [x] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [x] All changes manually tested?

- [x] Any documentation changes needed with this change?

- [x] Is the PR limited to 10 file changes?

- [x] Is the PR limited to one linear task?

- [x] Are relevant unit and component test-cases added in **new readability format**?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3896/test-report.html

**Review by sanpj2292 — APPROVED** [review]

**Review by koladilip — APPROVED** [review]

File: src/v0/util/index.test.js [hunk 1]
```diff
@@ -985,3 +987,65 @@ describe('removeHyphens', () => {
...
+    expect(result).toBe(expectedUuid);
+  });
+
+  test('should throw an error for empty input', () => {
+    const input = '';
+    expect(() => convertToUuid(input)).toThrow(InstrumentationError);
+    expect(() => convertToUuid(input)).toThrow('Input is empty or invalid.');
+  });
+
+  test('does not throw an error for null input', () => {
```
Comment (krishna2020) [inline src/v0/util/index.test.js:1021]: for undefined & null inputs, it should throw error

File: src/v0/util/index.js [hunk 2]
```diff
@@ -2330,6 +2331,24 @@ const isEventSentByVDMV1Flow = (event) => event?.message?.context?.mappedToDesti
 
 const isEventSentByVDMV2Flow = (event) =>
   event?.connection?.config?.destination?.schemaVersion === VDM_V2_SCHEMA_VERSION;
+
+const convertToUuid = (input) => {
+  const NAMESPACE = v5.DNS;
+  try {
```
Comment (krishna2020) [inline src/v0/util/index.js:2337]:
```
 // Check for null or undefined input
    if (input === null || input === undefined) {
      throw new InstrumentationError('Input cannot be null or undefined.');
    }
```

Comment (krishna2020) [reply src/v0/util/index.js:2337]: or use the functions we already have

Comment (shrouti1507) [reply src/v0/util/index.js:2342]: done

File: src/v0/util/index.test.js (hunk 1 above)
Comment (shrouti1507) [reply src/v0/util/index.test.js]: done

**Review by krishna2020 — APPROVED** [review]

**Pull request merged**